source/fundamentals/data-formats/custom-types.txt

```diff
@@ -361,22 +355,21 @@ We start by defining some arbitrary custom types:
       def __repr__(self):
           return "MyNumberType(%s)" % (self.__value,)
 
-We also define a fallback encoder that pickles whatever objects it receives
-and returns them as ``~bson.binary.Binary`` instances with a custom
-subtype. The custom subtype, in turn, allows us to write a TypeDecoder that
+You can define a fallback encoder that pickles the objects it receives
+and returns them as ``Binary`` instances with a custom
+subtype. This custom subtype allows you to define a ``TypeDecoder`` class that
```

Comment:
This is likely work for another ticket somewhere down the line, but I think this example can probably be simplified. As someone unfamiliar with "pickling" a lot of this is hard to parse for me, and the code here takes a big jump in complexity compared to the rest of our pages. Might be worth investigating sometime after the standardization project

Reply:
agreed